canonical-web-and-design/juju.is
# Build the about ops-code-quality page

## QA
- Go to http://0.0.0.0:8041/about/ops-code-quality
- Check the content matches the [copy doc[(https://docs.google.com/document/d/18_PRa2HA_P1j5JCbhZ0PsTRl9oVUHoeh6dI9u8GqQIs/edit#)

Fixes https://github.com/canonical-web-and-design/juju.is/issues/102
